.husky/pre-commit

```diff
@@ -1,3 +1,3 @@
 STAGED_ONLY=true npm run add-license
 npx lint-staged
-npm run test
+npm run test:unit
```

Comment:
The integration tests will start taking a long time once we add more tests. I was thinking we don't want to wait for the for every commit. Let me know if you have a preference.